pdns/pdnsutil.cc

```diff
@@ -913,10 +912,11 @@ static int increaseSerial(const DNSName& zone, DNSSECKeeper &dk)
 
   sd.db->startTransaction(zone, -1);
 
-  if (!sd.db->replaceRRSet(sd.domain_id, zone, rr.qtype, {rr})) {
-   sd.db->abortTransaction();
-   cerr<<"Backend did not replace SOA record. Backend might not support this operation."<<endl;
-   return -1;
+  vector<DNSResourceRecord> rrs(rr);
```

Comment:
```suggestion
  vector<DNSResourceRecord> rrs = {rr};
```

Reply:
why =?

Reply:
The initial syntax is wrong, there is no constructor in the `std::vector` class that accepts a single element. Now if your question is why ` = {rr}` and not `({rr})`, I don't really have an opinion since performance is completely irrelevant here, so whatever compiles would work for me.

Reply:
This is fixed.